middleware/envcheckmiddleware.php

```diff
@@ -46,6 +47,8 @@ class EnvCheckMiddleware extends CheckMiddleware {
 	private $environment;
 	/** @var IControllerMethodReflector */
 	protected $reflector;
+    /** @var IManager */
+    protected $shareManager;
```

Comment:
yes....i am correcting my ide settings.  Also i am trying to update unit tests...